### PR TITLE
Using checkers assert for integration-cli/docker_cli_events_unix_test.go

### PR DESCRIPTION
**Pull request by maaquib**
Part of #16756

Using checkers assert for integration-cli/docker_cli_events_unix_test.go

Signed-off-by: Mohammed Aaqib Ansari maaquib@gmail.com

### CODE REVIEW

File: integration-cli/docker_cli_events_unix_test.go [hunk 1]
```diff
@@ -22,35 +23,25 @@ func (s *DockerSuite) TestEventsRedirectStdout(c *check.C) {
 	dockerCmd(c, "run", "busybox", "true")
 
 	file, err := ioutil.TempFile("", "")
-	if err != nil {
-		c.Fatalf("could not create temp file: %v", err)
-	}
+	c.Assert(err, checker.IsNil, check.Commentf("could not create temp file: %v", err))
```
Comment (HuKeping) [inline integration-cli/docker_cli_events_unix_test.go]: Just `check.Commentf("could not create temp file")` is enough , the `checker.IsNil` will display the error if it not `nil`

File: integration-cli/docker_cli_events_unix_test.go [hunk 2]
```diff
@@ -22,35 +23,25 @@ func (s *DockerSuite) TestEventsRedirectStdout(c *check.C) {
...
 	defer os.Remove(file.Name())
 
 	command := fmt.Sprintf("%s events --since=%d --until=%d > %s", dockerBinary, since, daemonTime(c).Unix(), file.Name())
 	_, tty, err := pty.Open()
-	if err != nil {
-		c.Fatalf("Could not open pty: %v", err)
-	}
+	c.Assert(err, checker.IsNil, check.Commentf("Could not open pty: %v", err))
```
Comment (HuKeping) [inline integration-cli/docker_cli_events_unix_test.go]: same as above

File: integration-cli/docker_cli_events_unix_test.go [hunk 3]
```diff
@@ -63,43 +54,30 @@ func (s *DockerSuite) TestEventsOOMDisableFalse(c *check.C) {
 	go func() {
 		defer close(errChan)
 		out, exitCode, _ := dockerCmdWithError("run", "--name", "oomFalse", "-m", "10MB", "busybox", "sh", "-c", "x=a; while true; do x=$x$x$x$x; done")
-		if expected := 137; exitCode != expected {
-			errChan <- fmt.Errorf("wrong exit code for OOM container: expected %d, got %d (output: %q)", expected, exitCode, out)
-		}
+		expected := 137
```
Comment (vdemeester) [inline integration-cli/docker_cli_events_unix_test.go]: This one should not replace this one as it's used on next `select` with the chan `errchan`

File: integration-cli/docker_cli_events_unix_test.go [hunk 4]
```diff
@@ -111,43 +89,30 @@ func (s *DockerSuite) TestEventsOOMDisableTrue(c *check.C) {
 	go func() {
 		defer close(errChan)
 		out, exitCode, _ := dockerCmdWithError("run", "--oom-kill-disable=true", "--name", "oomTrue", "-m", "10MB", "busybox", "sh", "-c", "x=a; while true; do x=$x$x$x$x; done")
-		if expected := 137; exitCode != expected {
-			errChan <- fmt.Errorf("wrong exit code for OOM container: expected %d, got %d (output: %q)", expected, exitCode, out)
-		}
+		expected := 137
```
Comment (vdemeester) [inline integration-cli/docker_cli_events_unix_test.go]: Same here (about the chan)

File: integration-cli/docker_cli_events_unix_test.go [hunk 5]
```diff
@@ -111,43 +89,30 @@ func (s *DockerSuite) TestEventsOOMDisableTrue(c *check.C) {
...
+		c.Assert(createEvent[len(createEvent)-1], checker.Equals, "create", check.Commentf("event should be create, not %#v", createEvent))
+		c.Assert(attachEvent[len(attachEvent)-1], checker.Equals, "attach", check.Commentf("event should be attach, not %#v", attachEvent))
+		c.Assert(startEvent[len(startEvent)-1], checker.Equals, "start", check.Commentf("event should be start, not %#v", startEvent))
+		c.Assert(oomEvent[len(oomEvent)-1], checker.Equals, "oom", check.Commentf("event should be oom, not %#v", oomEvent))
 
 		out, _ = dockerCmd(c, "inspect", "-f", "{{.State.Status}}", "oomTrue")
-		if strings.TrimSpace(out) != "running" {
-			c.Fatalf("container should be still running, not %v", out)
-		}
+		c.Assert(strings.TrimSpace(out), checker.Equals, "running", check.Commentf("container should be still running, not %v", out))
```
Comment (vdemeester) [inline integration-cli/docker_cli_events_unix_test.go]: No need to print out — just "container should be still running".

**Comment by maaquib** [comment]
updated @HuKeping @vdemeester

**Comment by vdemeester** [comment]
LGTM 🐰

**Comment by cpuguy83** [comment]
LGTM

**Pull request merged**